static/templates/user_profile_modal.hbs

```diff
@@ -71,6 +71,7 @@
             </div>
 
             <div class="tabcontent" id="groups-tab">
+                <p style="padding-left:10px;"><a href="#organization/user-groups-admin" style="text-decoration:none" id="edit_user_group"><i class="fa fa-cog" aria-hdden="true"></i>  {{t 'Manage user groups'}} </a></p>
```

Comment:
We avoid inline `style` wherever possible; this should be done via an entry in a CSS rule.

Reply:
Also, `aria-hidden` has a typo, which I think would be more obvious written out in a CSS file (since everything on one line is hard to read).